client/src/glossary/SideBarActivator.js

```diff
@@ -0,0 +1,133 @@
+import React from "react";
+
+import { withRouter } from "react-router-dom";
+
+import { glossaryEntryStore } from "src/models/glossary";
+
+import { BackToTop } from "src/components";
+
+import { GlossaryMenu } from "./GlossaryMenu";
+
+const ROUTES_WITHOUT_GLOSSARY = {
+  "/start": true,
+  "/glossary": true,
+};
+
+const SidebarActivator = withRouter(
```

Comment:
The "activator" in the `TooltipActivator` came from it's (hacky) job of detecting and "activating" newly rendered tooltip targets, since we needed to run code to initialize them as they were inserted in to the DOM. Hard to give this a specific name because it's both the event handler for all glossary links AND the glossary menu itself at the same time. Maybe something like `GlossaryMenuSystem` to indicate that it's more than one thing ("system" isn't great, this is just a suggestion).